Pipfile

```diff
@@ -6,23 +6,23 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-certifi = "==2020.6.20"
-cffi = "==1.14.2"
-chardet = "==3.0.4"
-colorama = "==0.4.3"
-cryptography = "==3.3.2"
-idna = "==2.10"
-iso8601 = "==0.1.12"
-pycparser = "==2.20"
-python-dateutil = "==2.8.1"
-requests = "==2.24.0"
-securesystemslib = "==0.20.1"
-six = "==1.15.0"
+certifi = "*"
+cffi = "*"
+chardet = "*"
+colorama = "*"
+cryptography = "*"
+idna = "*"
+iso8601 = "*"
+pycparser = "*"
+python-dateutil = "*"
+requests = "*"
+securesystemslib = "*"
+six = "*"
 tuf = "==0.19.0"
-urllib3 = "==1.25.10"
-PyNaCl = "==1.4.0"
+urllib3 = "*"
+PyNaCl = "*"
```

Comment:
I would prefer we weaken the dependency specifications by providing floors or other bounded constraints on versions rather than unlocking the constraints entirely. If we need a lower minimum on some dependencies than today's, I understand, but it's risky for test reproducibility to unlock them entirely.

How about we bound them to versions that include (1) the explicit ones in `Pipfile` today and (2) the versions you found installed when you unlocked the constraints on macOS? That should fix your workflow today and provide an iterative path for resolving future issues without opening the barn doors fully.

The syntax for defining constraints in this file is specified in [PEP 440](https://peps.python.org/pep-0440/).

Reply:
@davidstrauss I've looked through that PEP and damned if I can find how to do an "or" type of constraint.

Reply:
I went ahead and did that. If you think that's acceptable, I think that would probably unblock things enough for now.